uktrade/data-hub-frontend
# Refactor Large Capital Profile tests and other cleanup

## Description of change

This is the fourth and final stage of the work started in #5480

This PR contains the following changes:
- Adding a new component to create a new LCP
- Restoring the functional tests commented out in #5480 and refactoring them
- Removal of all legacy code related to the Nunjucks implementation of this tab (this is the cause of the large diff)
- Some other minor refactoring to reduce duplication and ensure consistency across the new files

Once this PR has been merged I will open a new PR to merge the feature branch into `main`.

## Test instructions

Go to a company with no Large Capital Profile. It should be possible to create a new profile.

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
